### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by kk6axq**
Add instructions for launching.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# teleop_twist_keyboard
+Generic Keyboard Teleop for ROS
+#Launch
+To launch run: `rosrun teleop_twist_keyboard teleop_twist_keyboard.py`
```
Comment (trainman419) [inline README.md]: `launch` in ROS has a very specific meaning (launch files and roslaunch). This should just read: "To run:"

**Comment by kk6axq** [comment]
Could someone edit that part out. I can't seem to figure out how to do it.
Thanks,

**Comment by trainman419** [comment]
The pull request comes from your repository. You'll have to change it yourself.

**Comment by kk6axq** [comment]
The latest update to README.md removes the launch reference. Is this good?

**Comment by trainman419** [comment]
yes

**Pull request merged**